src/renderers/WebGLRenderer.js

```diff
@@ -2371,10 +2371,52 @@ class WebGLRenderer {
 
 		};
 
-		this.copyTextureToTexture = function ( position, srcTexture, dstTexture, level = 0 ) {
+		this.copyTextureToTexture = function ( srcTexture, dstTexture, srcRegion = null, dstPosition = null, level = 0 ) {
+
+			// support previous signature with source box first
```

Comment:
Shouldn't it be this? 
```
// support previous signature with dstPosition first
```

Reply:
Oh yeah - copy paste error

Reply:
https://github.com/mrdoob/three.js/commit/0e01940a39fab5d8d1fac0e8b10f0397fd665082